etcdserver/api/v3rpc/grpc.go

```diff
@@ -38,5 +39,7 @@ func Server(s *etcdserver.EtcdServer, tls *tls.Config) *grpc.Server {
 	pb.RegisterClusterServer(grpcServer, NewClusterServer(s))
 	pb.RegisterAuthServer(grpcServer, NewAuthServer(s))
 	pb.RegisterMaintenanceServer(grpcServer, NewMaintenanceServer(s))
+
+	go api.CapabilityLoop(s)
```

Comment:
If this doesn't belong in `etcdserver/api/v2http/client.go` it probably doesn't belong here either.